src/components/icons/actions/index.js

```diff
@@ -1,4 +1,8 @@
-export { default as ChevronRight } from './actions/ChevronRight.js';
-export { default as LocateOff } from './actions/LocateOff.js';
-export { default as LocateOn } from './actions/LocateOn.js';
-export { default as Search } from './actions/Search.js';
+export { default as checkmark } from './checkmark';
+export { default as chevron-right } from './chevron-right';
```

Comment:
Currently we have most stuff in camel case so you can copy a component name and paste it in a filename search, and uppercase names for components (with an exception for stuff where the code looks up pre-defined keys in JS objects). Is there a reason for the switch?

Reply:
@nero-still can you look at this?

Reply:
Fixed it, no additional work necessary here :)